cdhweb/pages/blocks/feature_block.py

```diff
@@ -26,7 +25,16 @@ class Meta:
             "ul",
         ],
     )
-    image = ImageBlock()
+    image = ImageChooserBlock(
+        label="Image",
+        required=True,
+    )
+
+    alt_text = blocks.CharBlock(
+        help_text="Alternative text in case the image can't be displayed.",
```

Comment:
I've seen this sentence elsewhere too. But that's not really what alt text is for (though it does do that). It's mainly for assistive technology (screen readers).

Reply:
ok no probs, got it from SK. Will update it